src/test/java/edu/harvard/iq/dataverse/EMailValidatorTest.java

```diff
@@ -1,48 +1,74 @@
 package edu.harvard.iq.dataverse;
 
 import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+import java.util.Collection;
+
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
 
+/**
+ *
+ * @author philip durbin
+ * @author oscardssmith
+ * @author rohit bhattacharjee
+ * @author stephen kraffmiller
+ * @author alex scheitlin
```

Comment:
Personally, I'm not a huge fan of these `@author` annotations. Netbeans adds them automatically when you create a new class. I always delete them. Other people on the team seem to either like them or don't bother to delete them. I don't know. There's no guidance on this at http://guides.dataverse.org/en/4.11/developers/coding-style.html so I'm fine with whatever for this pull request.

Reply:
Okey, I see. I added them because in the first class there was an `@author` tag too. I will not use the tag in the next commits. Thanks, for the feedback!